js/views/inbox_view.js

```diff
@@ -2,282 +2,282 @@
+      if (!options.initialLoadComplete) {
+        this.appLoadingScreen = new Whisper.AppLoadingScreen();
+        this.appLoadingScreen.render();
+        this.appLoadingScreen.$el.prependTo(this.el);
+        this.startConnectionListener();
+      }
 
-            this.inboxListView = new Whisper.ConversationListView({
-                el         : this.$('.inbox'),
-                collection : inboxCollection
-            }).render();
+      const inboxCollection = getInboxCollection();
 
-            this.inboxListView.listenTo(
-                inboxCollection,
-                'add change:timestamp change:name change:number',
-                this.inboxListView.updateLocation
-            );
-            this.inboxListView.listenTo(
-                inboxCollection,
-                'remove',
-                this.inboxListView.removeItem
-            );
+      inboxCollection.on('messageError', () => {
+        this.networkStatusView.render();
+      });
 
-            this.searchView = new Whisper.ConversationSearchView({
-                el    : this.$('.search-results'),
-                input : this.$('input.search')
-            });
+      this.inboxListView = new Whisper.ConversationListView({
+        el: this.$('.inbox'),
+        collection: inboxCollection,
+      }).render();
 
-            this.searchView.$el.hide();
+      this.inboxListView.listenTo(
+        inboxCollection,
+        'add change:timestamp change:name change:number',
+        this.inboxListView.updateLocation
+      );
+      this.inboxListView.listenTo(
+        inboxCollection,
+        'remove',
+        this.inboxListView.removeItem
+      );
 
-            this.listenTo(this.searchView, 'hide', function() {
-                this.searchView.$el.hide();
-                this.inboxListView.$el.show();
-            });
-            this.listenTo(this.searchView, 'show', function() {
-                this.searchView.$el.show();
-                this.inboxListView.$el.hide();
-            });
-            this.listenTo(this.searchView, 'open',
-                this.openConversation.bind(this, null));
+      this.searchView = new Whisper.ConversationSearchView({
+        el: this.$('.search-results'),
+        input: this.$('input.search'),
+      });
 
-            this.networkStatusView = new Whisper.NetworkStatusView();
-            this.$el.find('.network-status-container').append(this.networkStatusView.render().el);
+      this.searchView.$el.hide();
 
-            extension.windows.onClosed(function() {
-                this.inboxListView.stopListening();
-            }.bind(this));
+      this.listenTo(this.searchView, 'hide', function () {
+        this.searchView.$el.hide();
+        this.inboxListView.$el.show();
+      });
+      this.listenTo(this.searchView, 'show', function () {
+        this.searchView.$el.show();
+        this.inboxListView.$el.hide();
+      });
+      this.listenTo(
+        this.searchView, 'open',
```

Comment:
Too many parameters on one line!

Reply:
Yep. This was auto-fix auto-format based on previous code: https://github.com/signalapp/Signal-Desktop/blob/2a384cef7e29d6726cffc85d0ddda5fb85849e4a/js/views/inbox_view.js#L137-L138